overlay.nix

```diff
@@ -133,6 +118,8 @@ in rec {
   seth = self.callPackage (import ./src/seth) {};
   dapp = self.callPackage (import ./src/dapp) {};
 
+  dapp-tests = self.callPackage (import ./src/dapp-tests) {};
```

Comment:
I thought you wanted to remove this from the overlay?

Reply:
That was when we thought it would just be a shell script.

But because these integration tests are both dapp packages and other stuff (like a shell script), then I think it makes sense to use our solidity -> nix infrastructure, run tests with hevm and run anything else we want to run.